city_scrapers/spiders/wayne_building_authority.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# THIS SPIDER USES A MIXIN FOR SHARED FUNCTIONALITY.
+# MIXINS ARE STORED IN /city-scrapers/city-scrapers/mixins
+# YOU CAN OVERRIDE THE MIXIN HERE BY CREATING YOUR OWN DEFINITION.
+
+import re
+from datetime import datetime
+from dateutil.parser import parse as dateparse
+from city_scrapers.spider import Spider
+from city_scrapers.mixins.wayne_commission import Wayne_commission
+
+
+class Wayne_building_authoritySpider(Wayne_commission, Spider):
+    name = 'wayne_building_authority'
+    agency_id = 'Wayne County Building Authority'
+    start_urls = ['https://www.waynecounty.com/boards/buildingauthority/meetings.aspx']
+    meeting_name = 'Wayne County Building Authority'
+
+    # Override the mixin for any unique attributes.
+    location = {
+        'name': '6th Floor, Guardian Building',
+        'address': '500 Griswold St, Detroit, MI 48226',
+        'neighborhood': '',
+    }
+
+    def _parse_entries(self, response):
+        # There are several tables with old meetings. Look for the current year
+        # and get all rows which have children that contain text.
+        current_year = datetime.now().year
+        return response.xpath('//section[contains(.,"%s")]//tbody/tr[child::td/text()]' %current_year)
+
+    @staticmethod
+    def _parse_description(response):
+        """
+        Event description
+        """
+        desc = ("The County of Wayne will provide necessary reasonable "
```

Comment:
Based on the conversation today it would be good to just return an empty string here, which can probably just be done in the returned dict without using the `_parse_description` method